client/packages/invoices/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -99,12 +101,7 @@ export const OutboundShipmentListViewComponent: FC = () => {
 };
 
 export const OutboundShipmentListView: FC = () => (
-  <TableProvider
-    createStore={createTableStore}
-    queryParamsStore={createQueryParamsStore<OutboundRowFragment>({
-      initialSortBy: { key: 'otherPartyName' },
-    })}
```

Comment:
Fortunately the logic for this to be optional in TableProvider was already there, so this was an easy removal!

Reply:
the initial sort has been lost - looking at the original query

```
{
  "first": 20,
  "offset": 0,
  "key": "otherPartyName",
  "desc": false,
  "filter": {
    "type": {
      "equalTo": "OUTBOUND_SHIPMENT"
    }
  },
  "storeId": "D77F67339BF8400886D009178F4962E1"
}
```

this is now

```
{
  "first": 20,
  "offset": 0,
  "key": "status",
  "desc": false,
  "filter": {
    "type": {
      "equalTo": "OUTBOUND_SHIPMENT"
    }
  },
  "storeId": "D77F67339BF8400886D009178F4962E1"
}
```

I think we may need to pass in the initial sort, as well as the filter key

Reply:
Yup, good point. Fixed by passing in `initialSortByKey` as suggested. Thanks.